baram/iam_manager.py

```diff
@@ -61,3 +62,32 @@ def list_group_policies(self, user_group_name, max_items=100):
         :return:
         '''
         return self.cli.list_group_policies(GroupName=user_group_name, MaxItems=max_items)
+
+    def get_policies(self, max_result=1000):
+        """
+        Lists all policies in IAM
+        :return:
+        """
+        policies = self.cli.list_policies(MaxItems=max_result)
+        result = [i for i in policies['Policies']]
+        marker = policies['Marker']
+        while True:
+            next_policies = self.cli.list_policies(MaxItems=max_result, Marker=marker)
+            result += next_policies['Policies']
+            if 'Marker' in next_policies:
+                marker = next_policies['Marker']
+            else:
+                break
+        return result
```

Comment:
let me check this. 
1. Does `marker` always return value whether result has next page or not?
2. Does `marker` have only two cases? return empty string or filled string?

If so, I would implement this way.


```
policies = self.cli.list_policies(MaxItems=max_result)
result = policies['Policies']
while len(policies['Marker']) > 0:
    next_policies = self.cli.list_policies(MaxItems=max_result, Marker=policies['Marker'])
    result += next_policies['Policies']
    policies = next_policies
```

Reply:
1. If there is no next page, `Marker` parameter is not returned in the response.
2. So there are two cases: 1) there is `Marker` param with string value, 2) no `Marker` param in returned response.

Thank you for your suggestion. In similar context, I modified this part like below thanks to @Baescott 's help.
```
policies = self.cli.list_policies(Scope=scope, MaxItems=max_result)
        result = policies['Policies']
        try:
            while 'Marker' in policies:
                policies = self.cli.list_policies(Scope=scope, MaxItems=max_result, Marker=policies['Marker'])
                result += policies['Policies']
            return result
        except:
            self.logger.info('error')
```

Reply:
I think `try-except` is not bad approach to defense your code, but I recommend to apply all method with try-except or not.